### DISCUSSION TITLE
sharepoint挂载点，上传大文件卡住在"后端处理中"

### DISCUSSION THREAD

**Discussion by imxuebi** [AlistGo/alist#778] [Q&A]
sharepoint挂载点，上传大文件卡住在"后端处理中"

![Screenshot_20220318_202958_com microsoft emmx](https://user-images.githubusercontent.com/77877325/159003922-253642e5-7d08-4515-8607-7c9244a6a2d6.jpg)

**Comment by xhofe** [comment]
#413
